sos/report/plugins/block.py

```diff
@@ -50,14 +52,21 @@ def setup(self):
             "/sys/block/loop*/loop/",
         ])
 
-        cmds = [
-            "parted -s %(dev)s unit s print",
+        udevadm_cmds = [
             "udevadm info %(dev)s",
             "udevadm info -a %(dev)s"
         ]
-        self.add_device_cmd(cmds, devices='block', blacklist='ram.*')
+
+        parted_cmds = [
+            "parted -s %(dev)s unit s print"
+        ]
```

Comment:
If the list contains just one item, we can directly use the command `"parted -s .."` in the `add_device_cmd`, like we do for `fdisk -l ..` few lines below, no?

Reply:
agree. I will make the necessary changes.